stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -202,7 +201,7 @@ public fun MessageComposer(
 ) {
     val messageComposerState by viewModel.messageComposerState.collectAsState()
 
-    MessageComposer(
+    ChatTheme.componentFactory.MessageComposer(
```

Comment:
The entire message composer is now injected from the factories so that we can apply animation as part of the default behavior